manifests/r/RamSoft/DicomProxy/0.2.230-rbambhroliya-task-11558-Autopublish-dicomProxy.1/RamSoft.DicomProxy.yaml

```diff
@@ -0,0 +1,14 @@
+PackageIdentifier: "RamSoft.DicomProxy"
+PackageVersion: "0.2.230-rbambhroliya-task-11558-Autopublish-dicomProxy.1"
+PackageLocale: "en-US"
+Publisher: "RamSoft Inc."
+PackageName: "RamSoft DicomProxy"
+License: "Copyright © 1994-2021 RamSoft Inc. All rights reserved."
+ShortDescription: "DicomProxy is a windows appplication running as a windows service to assists local modalities to transfer dicom studies to a remote PACS server."
+Installers: 
+ - Architecture: "x64"
+   InstallerType: "exe"
+   InstallerUrl: "https://software.ramsoft.com/dicomproxy/DICOM-PROXY-0.2.230-rbambhroliya-task-11558-Autopublish-dicomProxy.1.EXE"
+   InstallerSha256: "2FEA46FAE58A8FF1F45B143440FD832D5A00F182ED832BE16D097F457F4350E3"
```

Comment:
```suggestion
   InstallerSha256: "32D9F632C111BDFD7702528C90C3EE8377A8FEE8C1CB65A6508FB2F6E21154C9"
```
I did another try today:
![image](https://user-images.githubusercontent.com/11600822/122395583-0a88f780-cf6f-11eb-8dd8-65c88750594d.png)

This is the hash I'm seeing when downloading it locally:
![image](https://user-images.githubusercontent.com/11600822/122395519-fa711800-cf6e-11eb-81c1-f50b6ffcbd25.png)

Reply:
@ItzLevvie  How do you get this sha256?